pkg/check/mysql_server.go

```diff
@@ -33,8 +33,22 @@ func NewMySQLVersionChecker(db *sql.DB, dbinfo *dbutil.DBConfig) Checker {
 	return &MySQLVersionChecker{db: db, dbinfo: dbinfo}
 }
 
-// MinVersion is mysql minimal version required
-var MinVersion = [3]uint{5, 5, 0}
+// SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
+// * 5.6.0 <= MySQL Version < 5.8.0
+// * 10.1.2 <= Mariadb Version
+var SupportedVersion = map[string]struct {
```

Comment:
s/SupportedVersion/DMSupportedVersion/ or put this in dm repo?

Reply:
if put it into dm repo, `check` would imports `dm`, `dm` would also import `check`; and it's also for syncer/checker tools